app/components/calls_to_action/mailinglist_block_component.rb

```diff
@@ -0,0 +1,22 @@
+module CallsToAction
+  class MailinglistBlockComponent < ViewComponent::Base
+    attr_reader :title, :image, :title_color, :text, :link_text, :link_target
+
+    def initialize(
+      title: "Get personalised updates",
+      image: "mailing-list-block-promo.jpg",
+      title_color: "yellow",
+      text: "Get free guidance and support on getting into teaching.",
```

Comment:
```suggestion
      text: "Sign up for free guidance and support on getting into teaching.",
```
Think there's a bit too much 'get' in this now.